examples/small-multiples/index.js

```diff
@@ -0,0 +1,87 @@
+const sales = [
+    {'Year':2011,'Month':'Jan','Sales':320},
+    {'Year':2011,'Month':'Feb','Sales':230},
+    {'Year':2011,'Month':'Mar','Sales':365},
+    {'Year':2011,'Month':'Apr','Sales':385},
+    {'Year':2011,'Month':'May','Sales':300},
+    {'Year':2012,'Month':'Jan','Sales':380},
+    {'Year':2012,'Month':'Feb','Sales':180},
+    {'Year':2012,'Month':'Mar','Sales':275},
+    {'Year':2012,'Month':'Apr','Sales':450},
+    {'Year':2012,'Month':'May','Sales':410},
+    {'Year':2013,'Month':'Jan','Sales':320},
+    {'Year':2013,'Month':'Feb','Sales':170},
+    {'Year':2013,'Month':'Mar','Sales':375},
+    {'Year':2013,'Month':'Apr','Sales':510},
+    {'Year':2013,'Month':'May','Sales':390},
+    {'Year':2014,'Month':'Jan','Sales':420},
+    {'Year':2014,'Month':'Feb','Sales':125},
+    {'Year':2014,'Month':'Mar','Sales':310},
+    {'Year':2014,'Month':'Apr','Sales':450},
+    {'Year':2014,'Month':'May','Sales':410},
+    {'Year':2015,'Month':'Jan','Sales':460},
+    {'Year':2015,'Month':'Feb','Sales':195},
+    {'Year':2015,'Month':'Mar','Sales':360},
+    {'Year':2015,'Month':'Apr','Sales':410},
+    {'Year':2015,'Month':'May','Sales':385}
+  ];
+  
+// group by month, giving our per-month small multiples
+const groupedByMonth = d3.nest()
+    .key(d => d.Month)
+    .entries(sales);
+
+// the various series components 
+const area = fc.seriesSvgArea()
+    .crossValue(d => d.Year)
+    .mainValue(d => d.Sales);
+const line = fc.seriesSvgLine()
+    .crossValue(d => d.Year)
+    .mainValue(d => d.Sales);
+const point = fc.seriesSvgPoint()
+    .crossValue(d => d.Year)
+    .mainValue(d => d.Sales);
+
+// the average line
+const average = fc.annotationSvgLine()
+    .value(d => d);
+
+// bring all these renderers together into one using the multi-series. The area/point/line
+// are mapped to the 'values' returned by d3.nest, and the line annotation maps to the average
+const multi = fc.seriesSvgMulti()
+    .series([area, line, point, average])
+    .mapping((data, index, series) => {
+        switch (series[index]) {
+        case average:
+            return [d3.mean(data.values.map(function(d) { return d.Sales; }))];
```

Comment:
Second argument to mean is an optional accessor. Probably also worth an arrow function

Reply:
good point - fixed